lab_2_tokenize_by_bpe/main.py

```diff
@@ -14,6 +15,16 @@ def prepare_word(
     :param end_of_word: a token that signifies the end of word
     :return: preprocessed word
     """
+    if (not isinstance(raw_word, str) or
+        not isinstance(start_of_word, str | None) or
+        not isinstance(end_of_word, str | None)):
+        return None
+    word = list(raw_word)
+    if start_of_word is not None:
+        word.insert(0, start_of_word)
+    if end_of_word is not None:
+        word.insert(len(word) + 1, end_of_word)
```

Comment:
append?

Reply:
rewowked

Reply:
Very good solution. But let's try to use `append`